### DISCUSSION TITLE
Issue labels `area-routing` vs. `feature-routing`

### DISCUSSION THREAD

**Discussion by gfoidl** [dotnet/aspnetcore#58839] [Q&A]
Which one to use? (the latter together with e.g. `area-minimal`).

If we are on `feature-routing` then the `area-routing` could be obsoleted.

@martincostello please share your thoughts.

**Comment by martincostello** [comment]
I would have thought the feature label would be the right one and that would be a child of some other area. But maybe we're both wrong 😄
